src/gitlab.js

```diff
@@ -158,7 +158,7 @@ export default class VerdaccioGitLab implements IPluginAuth {
   }
 
   allow_access(user: RemoteUser, _package: VerdaccioGitlabPackageAccess, cb: Callback) {
-    if (!_package.gitlab) { return cb(); }
+    if (!_package.gitlab) return cb();
```

Comment:
I'm not the decider of the style, but I would always keep braces in ifs, even in a single line, more maintanable option.

Reply:
We have been using this style in other projects. However I am also okay to bring it back if you both prefer that way.